### DISCUSSION TITLE
Vscode doesnt navigate to styles defined within createStyles

### DISCUSSION THREAD

**Discussion by intelseb** [mantinedev/mantine#1299] [Q&A] [answered]
Just wondering if the types are inferred correctly for createStyles, when creating styles and using them vscode won't navigate to the definition.

```
const useStyles = createStyles((theme) => ({
  description: {
    fontWeight: 700
  }
}))
```

e.g ```<p className={classes.description}/>``` when moving cursor onto description and clicking F12 I'd expect it to go straight to the description field within the createStyles function, i'm not sure if this is a limitation?

**Comment by rtivital** [comment, marked as answer]
I do not think that it is possible
